django_admin_bootstrapped/templates/admin/auth/user/change_password.html

```diff
@@ -1,5 +1,5 @@
 {% extends "admin/base_site.html" %}
-{% load i18n admin_urls admin_static admin_modify bootstrapped_goodies_tags %}
+{% load i18n admin_urls static admin_modify bootstrapped_goodies_tags %}
```

Comment:
The changes where `admin_static` were changed to `static` were done by me to get things working for Django 3.x